biggus/tests/test_integration.py

```diff
@@ -17,6 +17,7 @@
 from __future__ import absolute_import, division, print_function
 from six.moves import (filter, input, map, range, zip)  # noqa
 
+import mock
```

Comment:
Should be `from biggus.tests import mock`. I'm not sure why Travis has `mock` installed on Python 3; it's not supposed to be.